src/main/java/uk/gov/hmcts/reform/ccd/client/CaseAccessDataStoreApi.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.ccd.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import uk.gov.hmcts.reform.ccd.model.AddCaseAssignedUserRolesRequest;
+import uk.gov.hmcts.reform.ccd.model.AddCaseAssignedUserRolesResponse;
+import uk.gov.hmcts.reform.ccd.model.CaseAssignedUserRolesRequest;
+import uk.gov.hmcts.reform.ccd.model.CaseAssignedUserRolesResource;
+
+import java.util.List;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi.SERVICE_AUTHORIZATION;
+
+@FeignClient(name = "ccd-access-data-store-api", url = "${core_case_data.api.url}",
+    configuration = CoreCaseDataConfiguration.class)
+public interface CaseAccessDataStoreApi {
```

Comment:
Should this be part of https://github.com/hmcts/ccd-client library?

Reply:
may try later, ccd client is not stable at the moment

Reply:
let's create a tech debt story and we can merge this PR.